test/Ocelot.UnitTests/DependencyInjection/ConfigurationBuilderExtensionsTests.cs

```diff
@@ -280,7 +302,7 @@ private void ThenTheConfigsAreMergedAndAddedInApplicationConfiguration(bool useC
             fc.Routes.ShouldContain(x => x.UpstreamHost == (useCombinedConfig ? _combinedFileConfiguration.Routes[1].UpstreamHost : _routeB.Routes[0].UpstreamHost));
             fc.Routes.ShouldContain(x => x.UpstreamHost == (useCombinedConfig ? _combinedFileConfiguration.Routes[2].UpstreamHost : _routeB.Routes[1].UpstreamHost));
 
-            fc.Aggregates.Count.ShouldBe(useCombinedConfig ? _combinedFileConfiguration.Aggregates.Count :_aggregate.Aggregates.Count);
+            fc.Aggregates.Count.ShouldBe(useCombinedConfig ? _combinedFileConfiguration.Aggregates.Count : _aggregate.Aggregates.Count);

```

Comment:
Fake change

Reply:
You're right, I have format-on-save enabled, so this was just Visual Studio fixing whitespace issues. I'll revert this chunk.

Reply:
Whitespace removed.